docs/installation/integrations/react.md

```diff
@@ -338,11 +336,36 @@ class App extends Component {
 export default App;
 ```
 
+### Vite
+
+Vite requires linked packages to be ESM, and unfortunately, the CKEditor build is not ESM yet (but we're working on it). Therefore, you must modify the `vite.config.js` file to integrate a custom build with Vite. The snippet below will allow you to include the custom build in a Vite bundle. Check out the [Vite docs](https://vitejs.dev/guide/dep-pre-bundling.html#monorepos-and-linked-dependencies) to see more details.
+
+```js
+// vite.config.js
+
+export default defineConfig({
+  optimizeDeps: {
+    include: ['@workspace/ckeditor5-custom-build'],
+  },
+  build: {
+    commonjsOptions: {
+      include: [/@workspace\/ckeditor5-custom-build/, /node_modules/],
+    }
+  }
+})
+```
+
+Also, remember to use the correct import style for the CKEditor React component:
+
+```js
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+```
```

Comment:
I don't think we need this. It seemed like a minor oversight by the developer who reported this issue and the documentation above is showing this import correctly.

Reply:
I wasn't sure about it either. I'll remove it.